localstack/services/apigateway/helpers.py

```diff
@@ -722,7 +730,10 @@ def get_gateway_responses(api_id):
     region_details = APIGatewayRegion.get()
     result = region_details.gateway_responses.get(api_id, [])
 
-    href = "http://docs.aws.amazon.com/apigateway/latest/developerguide/restapi-gatewayresponse-{rel}.html"
+    href = (
+        "http://docs.aws.amazon.com/apigateway/latest/developerguide/restapi-gatewayresponse-{"
```

Comment:
nitpick: I understand that this is is done automatically by the code linter (limiting the max line length), but could we try to re-formatting these strings, so they look a bit more structured / less arbitrarily cut off? 😄 

For example:
```
         "http://docs.aws.amazon.com/apigateway/latest/developerguide/"
         "restapi-gatewayresponse-{rel}.html"
```

(same for `PATH_REGEX_TEST_INVOKE_API` above)